### PR TITLE
RNA pipeline design: Short But Wide

### PR DESCRIPTION
**Pull request by J35P312**
Aloha!
I have modified the RNA pipeline analysis recipie to run star-fusion from fastq files. I ran dry-run and it looks nice!

STAR-Fusion --genome_lib_dir /mnt/hds/proj/cust002/jesper/clinical_genomics_repo/MIP/skrot/mip_references --right_fq /scratch/$SLURM_JOB_ID/1_171111_H57VNCCXY-11_118-2-1U_CAGATCTG_1.fastq.gz --left_fq /scratch/$SLURM_JOB_ID/1_171111_H57VNCCXY-11_118-2-1U_CAGATCTG_2.fastq.gz --output_dir /mnt/hds/proj/cust002/jesper/clinical_genomics_repo/MIP/skrot/118/analysis/118-2-1U/star

### CODE REVIEW

File: lib/MIP/Recipes/Analysis/Star_fusion.pm [hunk 1]
```diff
@@ -43,6 +44,7 @@ sub analysis_star_fusion {
 ##          : $family_id               => Family id
 ##          : $file_info_href          => File_info hash {REF}
 ##          : $genome_lib_dir_path     => Path to the directory containing the genome library
+##          : infiles_ref              => \@{ $infile_href->{$sample_id} },
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Star_fusion.pm]: Should it not say "Infiles per sample id" here

File: lib/MIP/Recipes/Analysis/Star_fusion.pm [hunk 2]
```diff
@@ -189,71 +208,159 @@ sub analysis_star_fusion {
...
-        $infile_path = catfile( $insample_directory,
-            $infile_prefix . $infile_star_aln_prefix . $DOT . $infile_suffix );
+        # Collect interleaved info
+        my $interleaved_fastq_file =
+          $sample_info_href->{sample}{$sample_id}{file}{$infile_prefix}
+          {interleaved};
 
         ## Creates program directories (info & programData & programScript), program script filenames and writes sbatch header
-        my ( $file_path, $program_info_path ) = setup_script(
+        my ( $file_name, $program_info_path ) = setup_script(
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Star_fusion.pm]: Is it a file name or the file path that is returned?

**Comment by J35P312** [comment]
Now I have changed the $file_path variable!

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**